app/javascript/packs/formatted-fields.js

```diff
@@ -0,0 +1,23 @@
+import Cleave from 'cleave.js';
+
+const SELECTOR_CONFIGS = {
+  '.personal-key': {
+    blocks: [4, 4, 4, 4],
+    delimiter: '-',
+  },
+  '.backup-code': {
+    blocks: [4, 4, 4],
+    delimiter: '-',
+  },
+  '.zipcode': {
+    numericOnly: true,
+    blocks: [5, 4],
+    delimiter: '-',
+    delimiterLazyShow: true,
+  },
+};
+
+Object.entries(SELECTOR_CONFIGS)
+  .map(([selector, config]) => [document.querySelector(selector), config])
+  .filter(([element]) => element)
+  .forEach(([element, config]) => new Cleave(element, config));
```

Comment:
To confirm: The `Cleave` constructor can only handle one element, even though a class selector _could_ match multiple elements. This is true regardless if we pass the selector (previous) or element (current) to the constructor.

See: https://github.com/nosir/cleave.js/blob/e3fa6f3/src/Cleave.js#L29-L36

If we ever need to support multiple instances of elements on the page, we could consider constructing each:

```suggestion
Object.entries(SELECTOR_CONFIGS)
  .map(([selector, config]) => [document.querySelectorAll(selector), config])
  .filter(([elements]) => elements.length)
  .forEach(([elements, config]) => elements.forEach((element) => new Cleave(element, config)));
```